### PR TITLE
tablet design for Closing Account

### PR DESCRIPTION
**Pull request by utkarsha-deriv**
## Changes:

-replace old mixins with new

### Screenshots:

Please provide some screenshots of the change.
Screenshot 2024-05-23 at 5 53 10 PM

### CODE REVIEW

**Review by matin-deriv — APPROVED** [review]

**Review by fasihali-deriv — APPROVED** [review]

**Review by suisin-deriv — APPROVED** [review]

File: packages/account/src/Styles/account.scss [hunk 1]
```diff
@@ -1611,7 +1611,7 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 3%;
             list-style-type: disc;
 
-            @include mobile {
+            @include mobile-or-tablet-screen {
                 padding-left: 5%;
```
Comment (amina-deriv) [inline packages/account/src/Styles/account.scss]: @suisin-deriv @utkarsha-deriv shouldnt this be `padding-inline-start` ?

File: packages/account/src/Styles/account.scss [hunk 2]
```diff
@@ -1611,7 +1611,7 @@ $MIN_HEIGHT_FLOATING: calc(
             padding-left: 3%;
```
Comment (amina-deriv) [inline packages/account/src/Styles/account.scss]: can you check rtl layout here?

File: packages/account/src/Styles/account.scss [hunk 3]
```diff
@@ -1651,7 +1651,7 @@ $MIN_HEIGHT_FLOATING: calc(
             margin-inline-end: 10px;
             float: right;
```
Comment (amina-deriv) [inline packages/account/src/Styles/account.scss:1653]: @suisin-deriv shouldnt this be   float: inline-start;

Comment (suisin-deriv) [reply packages/account/src/Styles/account.scss:1653]: yes it should be 🙏

Comment (suisin-deriv) [reply packages/account/src/Styles/account.scss:1653]: @utkarsha-deriv can you help check it in RTL see if it's looking weird 🙏

File: packages/account/src/Styles/account.scss (hunk 1 above)
Comment (utkarsha-deriv) [reply packages/account/src/Styles/account.scss]: Yes, let me update this.

File: packages/account/src/Styles/account.scss (hunk 3 above)
Comment (utkarsha-deriv) [reply packages/account/src/Styles/account.scss:1653]: Sure, checking it 🙏

Comment (utkarsha-deriv) [reply packages/account/src/Styles/account.scss:1653]: Should it not be float: inline-end; @suisin-deriv @amina-deriv

**Review by amina-deriv — APPROVED** [review]

**Pull request merged**